dbt/seeds/etl_full_row_counts.csv

```diff
@@ -48,6 +48,23 @@ _core_eia923__monthly_cooling_system_information,2021,15080
 _core_eia923__monthly_cooling_system_information,2022,14676
 _core_eia923__monthly_cooling_system_information,2023,14383
 _core_eia923__monthly_cooling_system_information,2024,11376
+_core_eia923__yearly_byproduct_disposition,2008,7600
+_core_eia923__yearly_byproduct_disposition,2009,7590
+_core_eia923__yearly_byproduct_disposition,2010,7600
+_core_eia923__yearly_byproduct_disposition,2011,7570
+_core_eia923__yearly_byproduct_disposition,2012,7440
+_core_eia923__yearly_byproduct_disposition,2013,7380
+_core_eia923__yearly_byproduct_disposition,2014,7400
+_core_eia923__yearly_byproduct_disposition,2015,7210
+_core_eia923__yearly_byproduct_disposition,2016,6970
+_core_eia923__yearly_byproduct_disposition,2017,7650
+_core_eia923__yearly_byproduct_disposition,2018,7260
+_core_eia923__yearly_byproduct_disposition,2019,7080
+_core_eia923__yearly_byproduct_disposition,2020,6900
+_core_eia923__yearly_byproduct_disposition,2021,11870
+_core_eia923__yearly_byproduct_disposition,2022,4194
+_core_eia923__yearly_byproduct_disposition,2023,4170
+_core_eia923__yearly_byproduct_disposition,2024,3529
```

Comment:
Is there an explanation for why the row counts spike and then crash to a lower level here after more than a decade of consistency?

Reply:
Ah okay. Especially if they're moving toward not listing the plants with nothing to report int he current data, getting rid of those functionally null records from the past seems like it would make the table more legible and consistent over time.

What does it look like if we drop any records that are entirely null, or the only piece of "data" is the "no byproducts to report" column being "Y"? Doing this:

```python
byproducts.loc[byproducts.filter(like="tons_or_mmbtu").notna().any(axis="columns")]
```

Removes about 90% of all records. A lot of this is just most byproducts don't apply to most plants. Would we lose something valuable by doing this?

Reply:
Okay, the first one seems like a no regrets, non-destructive option -- there's truly no data of any kind for the whole plant in the table for that year and will hopefully result in comparable results across all years.

Maybe we can look at the 2nd option again if/when we come back to make an output table or deal with the mess of the `no_byproducts_to_report` column.